chiselFrontend/src/main/scala/chisel3/core/BlackBox.scala

```diff
@@ -80,7 +80,7 @@ abstract class ExtModule(val params: Map[String, Param] = Map.empty[String, Para
     component
   }
 
-  private[core] def initializeInParent() {
+  private[core] def initializeInParent(instanceCompileOptions: CompileOptions): Unit = {
```

Comment:
Would it make more sense to call this parentCompileOptions? (and similarly elsewhere where this is used)

Reply:
yeah I think parentCompileOptions is better, I'll change it